app/views/lessons/index.html.erb

```diff
@@ -24,7 +24,9 @@
             <td data-title="Date"><%= lesson.created_at.to_date.strftime("%B %d, %Y")%></td> 
             <% if @flags.pluck(:flagable_id).include? lesson.searchable_id %>
               <% flag = @flags.select{|x| x.flagable_id == lesson.searchable_id}[0] %>
-              <td> <a href="#" data-toggle="tooltip" title= "<%= flag.description %>">Blocked</a></td>    
+              <td>
+                <button class="tooltip_button" data-balloon-pos="left" data-balloon-length="medium" aria-label="<%= flag.description %>" data-balloon-pos="up">Blocked</button>
```

Comment:
Overall this definitely helps the visibility, especially the balloon on the left.  Unfortunately the row loses the ability to click on it and direct to the edit page, also the normal background doesn't exactly the background of the row (depending which one it is). I think an ideal fix would be to restore the clickability of the row and maybe add inline styling to make the background of the button transparent.

Reply:
a5e934bb3d24cf96c6b205063575dbc868b8aaae

Reply:
Went back to the previous solution with JS and changed the placement for the tooltip. Lets hope we get that working or more visible on mobile ??